extensions-contrib/orc-extensions/src/main/java/io/druid/data/input/orc/OrcHadoopInputRowParser.java

```diff
@@ -132,6 +137,22 @@ private List getListObject(ListObjectInspector listObjectInspector, Object listO
     return list;
   }
 
+  private void getMapObject(String parentName, MapObjectInspector mapObjectInspector, Object mapObject, Map<String, Object> parsedMap)
+  {
+    if (mapObjectInspector.getMapSize(mapObject) < 0) {
+      return;
+    }
+    String prefix = parentName + "_";
```

Comment:
What do you think about allowing the user to provide a format string that defines how the map field name and the subkey names are combined? Could be useful for avoiding potential field name collisions.

Reply:
Definitely would be useful. I could think of single handling for all map-types for now though. E.g.
```
"parser" : {
  "mapTypeFieldNameResolver": "{PARENT}_{KEY}"
}
```
Would you agree?

Reply:
Sounds good, I think a single format for all map types is fine